app/controllers/v1/tasks_controller.rb

```diff
@@ -0,0 +1,59 @@
+module V1
+  class TasksController < ApplicationController
+    swagger_controller :tasks, 'Tasks'
+
+    swagger_api :index do
+      summary 'List all tasks for a project'
+      notes 'This lists all tasks on a project'
+      param :query, :project_id, :uuid, :required, 'project reference'
+      param :query, :page, :integer, :optional, 'page number of results, default 1'
+      param :query, :page_size, :integer, :optional, 'number of results per page, default 25'
+    end
+    def index
+      tasks, errors = ListTasks.new(index_params).call
+      if errors.any?
+        render json: { errors: errors }, status: 400
+      else
+        render json: tasks
+      end
+    end
+
+    swagger_api :show do
+      summary 'Fetch a single Task'
+      param :path, :id, :string, :required, 'Task Id'
+    end
+    def show
+      task = Task.find params[:id]
```

Comment:
Using `find` here will raise an `ActiveRecord::RecordNotFound` error and your if/else logic will not be reached. You can choose to let Rails handle the not found exception with a 404 (standard behavior) in which case you don't need the logic, or you can use your own logic like you have here but use `find_by` which will return `nil` if not found.